### ISSUE TITLE
New title for files app 

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#2078]
The title for the files app is `Dashboard - #{portal}` where really it should be something like `File browsing - #{portal} - #{directory}`
